src/contract/event.rs

```diff
@@ -247,53 +247,29 @@ impl<T: Transport, E: Detokenize> Stream for EventStream<T, E> {
     }
 }
 
-/// The default block page size used for querying past events.
-pub const DEFAULT_BLOCK_PAGE_SIZE: u64 = 10_000;
-
 /// A builder for creating a filtered stream for any contract event.
 #[must_use = "event builders do nothing unless you stream them"]
 pub struct AllEventsBuilder<T: Transport, E: ParseLog> {
-    /// The underlying web3 instance.
     web3: Web3<T>,
-    /// The web3 filter builder used for creating a log filter.
-    filter: FilterBuilder,
-    /// The block to start retrieving logs from.
-    ///
-    /// This needs to be stored to work around the fact that the web3 filter
-    /// does not allow access to these values once stored.
-    pub from_block: Option<BlockNumber>,
-    /// The last block to retrieve logs for.
-    pub to_block: Option<BlockNumber>,
-    /// The topic filters that are encoded based on the event ABI.
-    pub topics: TopicFilter,
-    /// The polling interval for querying the node for more events.
-    pub poll_interval: Option<Duration>,
+    /// The underlying log filter for these contract events.
+    pub filter: LogFilterBuilder<T>,
```

Comment:
By the looks of it, all the removed attributes are now part of `FilterBuilder`. Is this correct?